reflex/components/core/cond.py

```diff
@@ -180,3 +181,20 @@ def cond(condition: Any, c1: Any, c2: Any = None):
         _var_full_name_needs_state_prefix=False,
         merge_var_data=VarData.merge(*var_datas),
     )
+
+
+def color_mode_cond(light: Any, dark: Any = None) -> Var | Component:
```

Comment:
This could go into `colormode.py` also

Reply:
i specifically moved it out here so it could be shared by chakra and radix or used standalone with any component library. `colormode.py` is radix specific